### DISCUSSION TITLE
meta viewport tag for live preview outer container

### DISCUSSION THREAD

**Discussion by martinbannister** [phcode-dev/phoenix#1460] [Q&A]
Could a `meta viewport` tag be added to the live preview container, please?  This would allow Chrome dev tools responsive device size testing to work, which it currently doesn't.  At present the whole browser window has to be resized to see a site's responsiveness change (unless I manually add a `viewport` `meta` tag after loading the preview).

**Comment by abose** [comment]
Just try out the popout live preview feature. That will popout the html into its own chrome tab where the dev tools responsive features should work as in any other web page. Will that help?

Im not sure of the workflow with the meta tag. Can you explain a bit more on that.

**Reply by abose** [reply]
I have not used the meta viewport feature. What is the meta tag that needs to be added, i will investigate based on that.
